src/zjs_util.h

```diff
@@ -275,4 +275,66 @@ int zjs_validate_args(const char *expectations[], const jerry_length_t argc,
         (zjs_validate_args((const char *[]){ __VA_ARGS__, NULL }, argc, argv) \
                 <= ZJS_INVALID_ARG) ? 1 : 0
 
+#define ZJS_FIND_NODE(type, list, cmp_element, cmp_to) \
```

Comment:
How about add a comment block above defining a sample struct like:

```c
// Type definition for examples below:
// struct mylistitem {
//     struct mylistitem *next;
//     int myvalue;
// } mylistitem_t;
// mylistitem_t *mylist = NULL;
```

Variable names like (type, list, field_name, value) might be clearer?

And then above each macro show a one-line example like:

```c
/**
 * Returns node from list with field field_name matching value, or NULL
 *
 * (Uses simple equality, will not work for string fields.)
 *
 * Example:
 *   mylistitem_t *item = ZJS_FIND_NODE(mylistitem_t, mylist, myvalue, 42);
*/
```